test/support/stub_model_filters.rb

```diff
@@ -0,0 +1,5 @@
+module StubModelFilters
+  def stub_filter(name)
+    define_singleton_method(name) { |*| nil }
+  end
+end
```

Comment:
Like that? https://github.com/plataformatec/devise/commit/e966ba72b1e663829aa16d2c41b336d1b6f37d35